src/sqlite3gen.cpp

```diff
@@ -871,15 +1419,132 @@ static void writeTemplateList(const ClassDef *cd)
 {
   writeTemplateArgumentList(cd->templateArguments(),cd,0);
 }
+
+QCString getSQLDocBlock(const Definition *scope,
+  const Definition *def,
+  const QCString &doc,
+  const QCString &fileName,
+  int lineNr)
+{
+  QGString s;
+  if (doc.isEmpty()) return s.data();
+  FTextStream t(&s);
+  DocNode *root = validatingParseDoc(
+    fileName,
+    lineNr,
+    const_cast<Definition*>(scope),
+    const_cast<MemberDef*>(reinterpret_cast<const MemberDef*>(def)),
+    doc,
+    FALSE,
+    FALSE
+  );
+  XMLCodeGenerator codeGen(t);
+  // create a parse tree visitor for XML
+  XmlDocVisitor *visitor = new XmlDocVisitor(t,codeGen);
+  root->accept(visitor);
+  delete visitor;
+  delete root;
+  QCString result = convertCharEntitiesToUTF8(s.data());
+  return result.data();
+}
+
+static void getSQLDesc(SqlStmt &s,const char *col,const char *value,const Definition *def)
+{
+  bindTextParameter(
+    s,
+    col,
+      getSQLDocBlock(
+        def->getOuterScope(),
+        def,
+        value,
+        def->docFile(),
+        def->docLine()
+      ),
+      FALSE
+    );
+}
 ////////////////////////////////////////////
 
+/* (updated Sep 01 2018)
+DoxMemberKind and DoxCompoundKind (compound.xsd) gave me some
```

Comment:
I had to collect this just to reason through things as I worked. I think it's worth keeping around for a bit in case the core concern ('kind' overlap for member/compound types generalized in the def table) materializes. Decided to flag it in case you think it has broader value that would justify moving it somewhere more permanent?